### PR TITLE
feat: homepage added

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshot
Screenshot (148)
Screenshot (149)

### CODE REVIEW

**Review by bishnoimukesh — APPROVED** [review]
seems good to merge

File: index.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    <link
+      rel="stylesheet"
+      href="https://arcui.netlify.app/styles/padding.css"
+    />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Meck Store</title>
+  </head>
+
+  <body>
+    <!-- Navbar -->
```
Comment (xii-vi) [inline index.html:41]: _Remove comment if not needed_

**Review by xii-vi — APPROVED** [review]
_Just a tiny comment. After that, You can merge._

File: index.html (hunk 1 above)
Comment (devdeadviz) [reply index.html:41]: It will be good to know which part of code represents what, that's why I have mentioned them.

**Pull request merged**